huxley/www/js/components/RegistrationView.js

```diff
@@ -112,301 +116,187 @@ var RegistrationView = React.createClass({
 
   render: function() {
     var conference = this.context.conference;
+
+    //I feel like this (And a lot of others in this section) could 
+    //Be reduced with some kind of mapping,
+    //But there's no obvious way to map over Objects in JS so :/
+    var accountInformationErrors = {
```

Comment:
But I don't see the reason not to construct the objects as needed when they're passed to the child component